.gitignore

```diff
@@ -1 +1,11 @@
 target/
+
+# Metals / bloop
+.metals/
+.bloop/
+
+# Intellij
+.idea/
+
+#vscode
+.vscode/
```

Comment:
Let's not add this one at least, I may commit some repository-specific settings, like done [here](https://github.com/coursier/coursier/blob/796767f765f5ef7211252370e76fcfb1882734d7/.vscode/settings.json).